PART_7/StrainTransmission/Iterate_and_job.py

```
@@ -0,0 +1,186 @@
+		job_count = check_job_count()
+		while job_count >= 100:
+			time.sleep(300)
+			job_count = check_job_count()
+		subprocess.call("sbatch "+Script_n, shell=True)
+		#print(Script_n)
+def Script2_LinearModel_transmission():
+	SGBs_run = []
+	script = "2_MotherInfantTranmission_Association.R"
+	with open("SGB_babies.tsv") as Info:
+		for line in Info:
+			l = line.rstrip().split()
```

Ruff (0.14.8)

Ambiguous variable name: `l`

(E741)